### ISSUE TITLE
IINA is ignoring file with extension ".aif" when loading folder to playlist

### ISSUE THREAD

**Issue by KuangLei** [iina/iina#2895]
**System and IINA version:**

- macOS 10.15.3 (19D76)
- IINA 1.0.7-beta2 Build 116

### Case 1

**Steps to reproduce:**

1. Make a folder with some `.aif` files and some `.mp3`, `.flac`, `.aiff` files.
2. Open one of the `.mp3`, `.flac`, `.aiff`.

**Expected behavior:**

There should be both all the `.aif` files and all the other `.mp3`, `.flac`, `.aiff` files loaded in the playlist.

**Actual behavior:**

`.aif` files are not loaded into the playlist. Other files are fine.

### Case 2

**Steps to reproduce:**

1. Make a folder with some `.aif` files and some `.mp3`, `.flac`, `.aiff` files.
2. Open one of the `.aif`; Let's make it `I_am_opened.aif`.

**Expected behavior:**

All the `.aif` files including that opened `I_am_opened.aif` should be loaded into the playlist, and also all the other `.mp3`, `.flac`, `.aiff` files.

**Actual behavior:**

Only the `I_am_opened.aif`, together with other `.mp3`, `.flac`, `.aiff` files are loaded into the playlist. `.aif` files other than `I_am_opened.aif` is not loaded.

### Case 3

**Steps to reproduce:**

1. Make a folder with only some `.aif` files.
2. Open one of the `.aif`, `I_am_opened.aif`.

**Expected behavior:**

All the files should be loaded.

**Actual behavior:**

Only the exact file is loaded into the playlist.

**How often does this happen?**

Every time.

---

### In short

IINA should recognize `.aif` files alongside `.aiff` files when adding a folder to the playlist.

Because `.aif` extension is recognized as standard AIFF format. See page 7 of [this](http://www-mmsp.ece.mcgill.ca/Documents/AudioFormats/AIFF/Docs/AIFF-1.3.pdf) document. And Logic Pro X is using `.aif` for AIFF file by default. So I think there are no reason not to add `.aif` to the filter.

Thank you very much for developing this wondering player.
